core/server/master/src/main/java/alluxio/master/file/loadmanager/LoadManager.java

```diff
@@ -286,7 +287,12 @@ private boolean scheduleBatch(
     }
     LoadRequest request = buildRequest(
         batch, load.getBandWidth() / mActiveWorkers.size());
-    ListenableFuture<LoadResponse> responseFuture = workerClient.get().load(request);
+    ListenableFuture<LoadResponse> responseFuture;
+    try {
+      responseFuture = workerClient.get().load(request);
+    } catch (AlluxioStatusException e) {
+      throw e.toGrpcStatusRuntimeException();
+    }
```

Comment:
@rongrong we are refactoring `DefaultBlockWorkerClient` to throw checked exceptions. Do you know what's the appropriate way to handle the checked exceptions here?